wp-includes/integrations/class-wp-service-worker-styles-integration.php

```diff
@@ -12,6 +12,14 @@
  */
 class WP_Service_Worker_Styles_Integration extends WP_Service_Worker_Base_Integration {
 
+	/**
+	 * Scope this integration applies to.
+	 *
+	 * @since 0.2
+	 * @var int
+	 */
+	protected $scope = WP_Service_Workers::SCOPE_ALL;
```

Comment:
@felixarntz Note how this had to be added to the subclass to override the variable in the base class. Something doesn't seem quite right.

Reply:
@westonruter I think it makes sense to have integrations define their scope. However, the implementation I made may not be obvious enough. Currently, the default scope is `SCOPE_FRONT`, via the property. So in order to change it to `SCOPE_ADMIN` or `SCOPE_ALL` you need to reset the property in the child class. Two ideas:
* It might make more sense to set the default scope to `SCOPE_ALL`.
* Or alternatively, maybe it's better to not have any default scope, but instead make the `WP_Service_Worker_Base_Integration::get_scope()` method abstract, thus require each integration to explicitly define it. That would also make it obvious that you as the interface developer _have_ to handle it.

Reply:
This can be removed, as it is no longer necessary after #72.